Makefile

```diff
@@ -46,6 +46,7 @@ SWAGGER_GENERATE = swagger generate client \
 		--tags="Organization pipelines jobs" \
 		--tags="Organization pipelines jobs build" \
 		--tags="Organization projects" \
+		--tags="Organization Projects" \
```

Comment:
This fixes the issue with the current swagger having a typo in Organization Project category